mgr/src/cmds/helpers.cr

```diff
@@ -105,3 +107,23 @@ def yes(label)
   value = STDIN.gets(chomp: true).not_nil!
   ["yes", "y", "yy", "ok", "sure", "on"].includes?(value.strip.downcase)
 end
+
+def handle_json_output(data, args)
+  if args.json
+    if data && data != ""
+      puts data.to_pretty_json
+      exit 0
+    else
+      puts "{}".to_json
+      exit 0
+    end
+  end
+end
+
+def handle_json_error(message, args)
+  if args.json
+    err = {"error": message}.to_json
+    puts err
```

Comment:
You can remove temp var. Directly use `puts {"error": message}.to_json`

Reply:
It was showing unexpected ":" so used a var

Reply:
`puts ({"error": message}.to_json)`

Reply:
updated.